pkg/prom/instance/instance.go

```diff
@@ -189,6 +189,9 @@ func (c *Config) ApplyDefaults(global GlobalConfig) error {
 			return fmt.Errorf("found duplicate remote write configs with name %q", cfg.Name)
 		}
 		rwNames[cfg.Name] = struct{}{}
+
+		// Send exemplars by default
+		cfg.SendExemplars = true
```

Comment:
(Note that this means you can't ever disable it)

Reply:
Ideally it should enabled by default and be disable-able through config, right?

I couldn't find a clean way to do this. Specially because `RemoteWrite` is a slice, so it's harder to apply a default. If you have any thoughts on how to achieve this, let me know. If not, I think leaving it as false by default and apply the config manually will be best.

Reply:
Removed the default. We will need to explicitly enable it. But that is how prometheus works too.